admin/election_controller.go

```diff
@@ -0,0 +1,89 @@
+package admin
+
+import (
+	"encoding/json"
+
+	"github.com/google/uuid"
+	"github.com/vocdoni/blind-csp/model"
+	"github.com/vocdoni/blind-csp/types"
+	"go.vocdoni.io/dvote/httprouter"
+	"go.vocdoni.io/dvote/httprouter/bearerstdapi"
+)
+
+type ElectionController struct {
+	store model.ElectionStore
+}
+
+func NewElectionController(store model.ElectionStore) *ElectionController {
+	return &ElectionController{store: store}
+}
+
+func (c *ElectionController) Create(msg *bearerstdapi.BearerStandardAPIdata, ctx *httprouter.HTTPContext) error {
+	newElection := model.Election{}
+	if err := json.Unmarshal(msg.Data, &newElection); err != nil {
+		return err
+	}
+
+	election, err := c.store.Create(&newElection)
+	if err != nil {
+		return err
+	}
+
+	electionBearerToken := uuid.NewSHA1(uuid.NameSpaceDNS, []byte(election.ID.String()))
+	responseMap := map[string]interface{}{
+		"adminToken": electionBearerToken.String(),
+		"data":       election,
+	}
+
+	resp, err := json.MarshalIndent(responseMap, "", " ")
+	if err != nil {
+		return err
+	}
+
+	return ctx.Send(resp, bearerstdapi.HTTPstatusCodeOK)
+}
+
+func (c *ElectionController) Get(msg *bearerstdapi.BearerStandardAPIdata, ctx *httprouter.HTTPContext) error {
```

Comment:
Instead of getters names use the name, for example `Election()` and for the List method you could use `Elections()`, etc.